dev-ruby/websocket/websocket-1.1.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
```

Comment:
I assume you don't have ruby22 installed and thus not tested with it. Normally we try to cover as many ruby targets as possible. We can also handle this when it lands in the tree.